util/winutil/winutil_notwindows.go

```diff
@@ -9,6 +9,20 @@ package winutil
 
 const RegBase = ``
 
+// GetPolicyString looks up a registry value in our local machine's path for
+// system policies, or returns the given default if it can't.
+//
+// This function will only work on GOOS=windows. Trying to run it on any other
```

Comment:
rather than repeating docs over multiple files with multiple builds tags, I often prefer to put the exported func + docs in one portable file and then call an unexported one that differs per build tag file, which then doesn't need docs.

Reply:
Fixing